image-builder/builder-services/src/main/java/io/hyscale/builder/services/util/DockerImageUtil.java

```diff
@@ -74,5 +76,22 @@ public void pullImage(String imageName) throws HyscaleException {
             throw new HyscaleException(ImageBuilderErrorCodes.FAILED_TO_PULL_IMAGE, imageName);
         }
     }
+    
+    public String getImageName(ServiceSpec serviceSpec) {
+    	String image=null;
+    	String registry=null;
+    	try {
+			image = serviceSpec.get(HyscaleSpecFields.getPath(HyscaleSpecFields.image, HyscaleSpecFields.name),
+					String.class);
+			registry=serviceSpec.get(HyscaleSpecFields.getPath(HyscaleSpecFields.image, HyscaleSpecFields.registry),
```

Comment:
This utility is duplicated , please check Imageutil from service-spec-commons module.

You can use that util